### ISSUE TITLE
[RSS] Handle empty summary

### ISSUE THREAD

**Issue by Injabie3** [SFUAnime/Ren#185]
For the summary portion, handle the case where we extract nothing. See below:
```
[24/02/2019 22:26] INFO rss rss 216: Scanning feed(s) for updates...
[24/02/2019 22:26] INFO rss getFeed 196: 1 new items from http://feeds.feedburner.com/crunchyroll/animenews?format=xml
[24/02/2019 22:26] INFO rss getFeed 196: 2 new items from https://www.animenewsnetwork.com/all/rss.xml
[24/02/2019 22:26] INFO rss getFeed 194: No new items from https://myanimelist.net/rss/news.xml
[24/02/2019 22:26] ERROR rss rss 264: Could not post to RSS channel!
[24/02/2019 22:26] ERROR rss rss 265: BAD REQUEST (status code: 400)
[24/02/2019 22:26] ERROR rss rss 266: Embed: {'image': {'url': 'https://www.animenewsnetwork.com/thumbnails/crop600x315/cms/news.2/141198/never-learn-kv.jpeg'}, 'url': 'http://www.animenewsnetwork.com/daily-brie
fs/2019-02-24/we-never-learn-anime-premieres-on-april-6/.143855', 'footer': {'text': 'This update is from https://www.animenewsnetwork.com/all/rss.xml?ann-edition=us', 'icon_url': 'https://upload.wikimedia.org/w
ikipedia/en/thumb/4/43/Feed-icon.svg/1200px-Feed-icon.svg.png'}, 'type': 'rich', 'title': "'We Never Learn' Anime Premieres on April 6", 'fields': [{'name': 'Date Published', 'value': 'Sun, 24 Feb 2019 08:57PM',
 'inline': False}, {'name': 'Summary', 'value': '', 'inline': False}], 'color': 15105570}
```

**Comment by KaguneAstra** [comment]
Does this problem still occur on the V3 version?

**Comment by Injabie3** [comment]
Yes, blank summaries still cause this:
```
[2019-08-16 22:56:31] [ERROR] discord: Could not post to RSS channel!
[2019-08-16 22:56:31] [ERROR] discord: 400 BAD REQUEST (error code: 50035): Invalid Form Body
In embed.fields.1.value: This field is required
[2019-08-16 22:56:31] [ERROR] discord: Embed: {'footer': {'text': 'This update is from https://www.animenewsnetwork.com/all/rss.xml?ann-edition=us', 'icon_url': 'https://upload.wikimedia.org/wikipedia/en/thumb/4/43/Feed-icon.svg/1200px-Feed-icon.svg.png'}, 'image': {'url': 'https://www.animenewsnetwork.com/thumbnails/crop600x315/cms/daily-briefs/150140/grandia.jpg'}, 'fields': [{'inline': False, 'name': 'Date Published', 'value': 'Fri, 16 Aug 2019 10:02PM'}, {'inline':
False, 'name': 'Summary', 'value': ''}], 'color': 15105570, 'type': 'rich', 'url': 'http://www.animenewsnetwork.com/daily-briefs/2019-08-17/grandia-grandia-ii-hd-remasters-launch-for-pc-in-september/.150140', 'title': 'Grandia, Grandia II HD Remasters Launch for PC in September'}
```

**Comment by Injabie3** [comment]
Ignoring for v2.
v3 merged in as of commit cabb3584db5246056a94a17b70cfaea133d0ebbe, closing.
